src/algorithms/controllers/msort_arr_bup.js

```diff
@@ -198,11 +181,7 @@ export function run_msort() {
     let A = nodes;
     let B = [...entire_num_array].fill(undefined);
     let runlength = 1; // length of run to merge
-    let size = nodes.length - 1; // size of array
-
-    console.log("runlength = " + runlength);
-
-    let runlengthString = 'runlength = ' + runlength;
+    const size = nodes.length - 1; // size of array
```

Comment:
maybe change so that it is just `nodes.length` instead of `nodes.length - 1`, causes some confusioin in the code

Reply:
Problem resolved